### ISSUE TITLE
finetuned model returns garbage

### ISSUE THREAD

**Issue by jprobichaud** [jumon/whisper-finetuning#9]
Thansk for this work, it's really helping getting started!

I have few hundred hours of 5-15 secs utterances transcribed that I'd like to finetune whisper on.  I was able to generate the json files properly and run finetuning with the default params on the medium.en model.  

Running transcribe.py on the saved checkpoints, I see that the model is producing  pure garbage (I barely have meaningful sentences...) My audios are in wav, pcm, 16kHz.  

Is the default learning rate problematic?  Anything special to take into account?

**Comment by jumon** [comment]
Hi, thank you for giving this repository a try. I apologize that it did not work well for you. 

It is possible that this was due to inadequate hyper-parameters. Training neural networks is highly dependent on hyper-parameters, and finding the right hyper-parameters is also influenced by the specific dataset used. However, I was able to successfully fine-tune Whisper-medium on the CSJ (Japanese) dataset, resulting in a significant reduction in the Character Error Rate from 14.7% to 4.8%. Perhaps, you could try tuning the hyper-parameters to see if it improves the performance. 

Thank you again for giving this repository a shot.

**Comment by emcodem** [comment]
After reading all the info about how the models were trained, i suspect the problem here could be related to training "utterances" (if that means inclomplete parts of sentences). It seems to be like you get what you train, e.g. if you train single words then you get single words output but you would never get any punctuation in this case... Does that make sense? 

Further theory: Single word output like the name of a player in football game as standalone "sentence"  could be very much depending on the right prompt. 
E.g. 
[0] Chelsea attacking now. [1]
[5] Pulisic... [6]
[8] Goal, Goal! What a wonderful, unexpected goal by Pulisic [10]
